zeppelin-web-angular/src/app/share/header/header.component.html

```diff
@@ -67,7 +67,7 @@
           <li nz-menu-item (click)="logout()">Logout</li>
         </ng-container>
         <li nz-menu-divider></li>
-        <li nz-menu-item><a href="/">Old Version</a></li>
+        <li nz-menu-item><a href="/classic">Old Version</a></li>
```

Comment:
Perhaps “Classic Version” should be used here instead of “Old Version”.
```suggestion
        <li nz-menu-item><a href="/classic">Classic Version</a></li>
```